README.md

```diff
@@ -61,14 +61,205 @@ target:
+| HARBOR      | HELM        | :x:                |
+| HARBOR      | CHARTMUSEUM | :white_check_mark: |
+| HARBOR      | HARBOR      | :white_check_mark: |
+
+
+### Harbor example
+
+In the case of HARBOR kind repos, be aware that chart repository URLs are:
+
+https://$HARBOR_DOMAIN/chartrepo/$HARBOR_PROJECT
+
+So if HARBOR_DOMAIN=my.harbor.com and HARBOR_PROJECT=my-project, you would need to specify this repo in the config file like:
+
+~~~yaml
+target:
+ repo:
+   kind: HARBOR
+   url: "https://my.harbor.com/chartrepo/my-project"
+~~~
+
+## Requirements
+
+In order for this tool to be able to successfully migrate a chart from a source repository to another it must fulfill the following requirements:
+
+- If the chart has dependencies they are specified in a requirements.yaml file. Currently, if the dependencies are specified in the Chart.yaml file the tool won't be able to manage them.
+- The images used by the chart must be specified in the following way in the values.yaml file:
+
+~~~yaml
+image:
+  registry: docker.io
+  repository: bitnami/ghost
+  tag: 3.22.2-debian-10-r0
+~~~
+
+The parent section name does not matter. In the previous example, instead of `image` it could be `mainImage` or whatever other name.
+
+The important thing is that the image name is specified with `registry`, `repository` and `tag`.
+
+The values of the parameteres `containerRegistry` and `containerRepositories` from the configuration file will be used to update the `registry` and `repository` properties in the values.yaml. The `tag` remains unchanged.
+
+> :warning: Be aware that this tool expect the images to be already present in the target container registry.
+
+## Changes performed in a chart
+
+In order to migrate a chart from one repository to another and retrieve the images from a new container registry, this tools performs the following changes in the chart code:
+
+#### Update values.yaml and values-production.yaml (if exists)
+
+These files are updated with the new container registry where the chart should pull the images from.
+
+#### Update requirements.yaml and requirements.lock
+
+If the chart has any dependency, they should be registered in this file. The requirements.yaml and requirements.lock file will be updated to retrieve the dependencies from the target repository.
+
+#### Update README.md
+
+README files for bitnami charts includes a TLDR section with instructions to add the helm repository to helm cli and 
+
+------
+
+Let's see the performed changes with an example. Imagine I sync the Ghost chart from the Bitnami chart repo to a local chartmuseum repo with no authentication.
+
+I would use this config file:
+
+~~~yaml
+source:
+  repo:
+    kind: HELM
+    url: "https://charts.bitnami.com/bitnami"
+target:
+  containerRegistry: "my.registry.io"
+  containerRepository: "test"
+  repo:
+    kind: CHARTMUSEUM
+    url: "http://localhost:8080"
+~~~
+
+After executing the tool, these are the changes performed to the following files:
```

Comment:
can you add docs for devs so this example can be updated easily if we update the tool? (no need to do so here)

Reply:
do you mean documenting how to get these diffs? Ook, I think I can document the process